lib/libc/minimal/Kconfig

```diff
@@ -39,6 +39,9 @@ config MINIMAL_LIBC_TIME
 	bool "Time functions"
 	select COMMON_LIBC_TIME if POSIX_TIMERS
 	select COMMON_LIBC_GMTIME_R
+	select COMMON_LIBC_ASCTIME_R
+	select COMMON_LIBC_LOCALTIME_R_UTC
+	select COMMON_LIBC_CTIME_R
```

Comment:
Shouldn't something like these be added to a POSIX Kconfig option too?

`POSIX_THREAD_SAFE_FUNCTIONS` under `lib/posix/options/Kconfig.pthread`.